docs/DafnyRef/UserGuide.md

```diff
@@ -599,8 +852,8 @@ PROVER_OPTIONS="\
   "
 
 "$DOTNET" "$BOOGIE" $BOOGIE_OPTIONS $PROVER_OPTIONS "$@"
-# Uncomment if you want to use the executable instead of the DLL
-# "$BOOGIE" $BOOGIE_OPTIONS $PROVER_OPTIONS "$@"
+#Uncomment if you want to use the executable instead of the DLL
+#"$BOOGIE" $BOOGIE_OPTIONS $PROVER_OPTIONS "$@"
```

Comment:
?

Reply:
The space triggered the automatic numbering to complain that 2 sections were missing. Removing this space made the [regex not create a false match](https://github.com/dafny-lang/dafny/blob/master/docs/DafnyRef/Numbers.java#L16) anymore:

Reply:
THe `\h+ you mean?`That would be even worse, my workaround would not work anymore.

Reply:
No, the `h*` before the `#`

Reply:
But it doesn't really matter, let's forget about this for now ^^